### PR TITLE
fix(release-plan): update release plan form, add status and release plan addmision column

### PR DESCRIPTION
**Pull request by sahil143**
## Fixes 

https://issues.redhat.com/browse/KFLUXBUGS-572

## Description

update release plan form

## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

image
=============#############################============
image
=============##############################===========
image

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/components/ReleaseService/ReleasePlan/ReleasePlanForm/form-utils.ts [hunk 1]
```diff
@@ -108,7 +116,7 @@ export const createReleasePlan = async (
       application,
       ...(data ? { data } : {}),
       serviceAccount,
-      target: targetWs,
+      target: `${targetWs}-tenant`,
```
Comment (abhinandan13jan) [inline src/components/ReleaseService/ReleasePlan/ReleasePlanForm/form-utils.ts:119]: move this to const WORKSPACE_SUFFIX or something

**Comment by Katka92** [comment]
Hello, I took a look at your PR. I created a Release plan with the `in target workspace` pipeline and the `Auto release` turned off. After creation, the `Auto release` is **True**. It's turned on when I click `Edit release plan`. Even when I try to edit it and turn it off, it stays **True**. Is that intended? Should I create a separate issue for it? Thx!

**Comment by sahil143** [comment]
> Hello, I took a look at your PR. I created a Release plan with the `in target workspace` pipeline and the `Auto release` turned off. After creation, the `Auto release` is **True**. It's turned on when I click `Edit release plan`. Even when I try to edit it and turn it off, it stays **True**. Is that intended? Should I create a separate issue for it? Thx!

@Katka92 I have fixed this issue. Thanks for testing the PR.

**Pull request merged**